src/main/java/org/gridsuite/study/server/service/RootNetworkNodeInfoService.java

```diff
@@ -858,4 +857,54 @@ public void stopPccMin(UUID studyUuid, UUID nodeUuid, UUID rootNetworkUuid) {
+        RootNetworkNodeInfoEntity rootNetworkNodeInfoEntity = rootNetworkNodeInfoRepository.findAllByNodeExportNetworkExportUuid(exportUuid)
+                .stream()
+                .filter(rootNetworkNodeInfo -> rootNetworkNodeInfo.getNodeExportNetwork()
+                        .stream().anyMatch(nodeExportEmbeddable -> nodeExportEmbeddable.getExportUuid().equals(exportUuid)))
+                .findFirst()
+                .orElseThrow(() -> new StudyException(NOT_FOUND, "Root network not found for exportUuid=" + exportUuid));
+        rootNetworkNodeInfoEntity.getNodeExportNetwork().stream()
+                .filter(nodeExportEmbeddable -> nodeExportEmbeddable.getExportUuid().equals(exportUuid))
+                .findFirst()
+                .ifPresentOrElse(nodeExportEmbeddable -> nodeExportEmbeddable.setStatus(status),
+                        () -> rootNetworkNodeInfoEntity.getNodeExportNetwork().add(toNodeExportEmbeddable(exportUuid, status)));
+    }
+
+    @Transactional
+    public void updateExportNetworkStatus(UUID nodeUuid, UUID rootNetworkUuid, UUID exportUuid, ExportNetworkStatus status) {
+        RootNetworkNodeInfoEntity rootNetworkNodeInfoEntity = findRootNetworkNodeInfo(nodeUuid, rootNetworkUuid);
+        rootNetworkNodeInfoEntity.getNodeExportNetwork().stream()
+                .filter(e -> e.getExportUuid().equals(exportUuid))
+                .findFirst()
+                .ifPresentOrElse(nodeExportEmbeddable -> nodeExportEmbeddable.setStatus(status),
+                        () -> rootNetworkNodeInfoEntity.getNodeExportNetwork().add(toNodeExportEmbeddable(exportUuid, status)));
+    }
+
+    @Transactional
+    public ExportNetworkStatus getExportNetworkStatus(UUID nodeUuid, UUID rootNetworkUuid, UUID exportUuid) {
+        RootNetworkNodeInfoEntity rootNetworkNodeInfoEntity = findRootNetworkNodeInfo(nodeUuid, rootNetworkUuid);
+        return rootNetworkNodeInfoEntity.getNodeExportNetwork().stream()
+                .filter(nodeExportEmbeddable -> nodeExportEmbeddable.getExportUuid().equals(exportUuid))
+                .map(NodeExportEmbeddable::getStatus)
+                .findFirst()
+                .orElseThrow(() -> new StudyException(NOT_FOUND, "Export network not found for exportUuid=" + exportUuid));
+    }
+
+    private RootNetworkNodeInfoEntity findRootNetworkNodeInfo(UUID nodeUuid, UUID rootNetworkUuid) {
+        return rootNetworkNodeInfoRepository.findByNodeInfoIdAndRootNetworkId(nodeUuid, rootNetworkUuid)
+                .orElseThrow(() -> new StudyException(NOT_FOUND, "Root network not found"));
+    }
+
+    @Transactional
+    public void clearNodeExportNetworks(UUID nodeUuid, UUID rootNetworkUuid) {
+        RootNetworkNodeInfoEntity rootNetworkNodeInfo = findRootNetworkNodeInfo(nodeUuid, rootNetworkUuid);
+        if (rootNetworkNodeInfo.getNodeExportNetwork() != null) {
+            rootNetworkNodeInfo.getNodeExportNetwork().clear();
+        } else {
```

Comment:
Remove this branch ? It's not in the clearNodeExportNetwork that we should set a new ArrayList<> instead of a null